src/js/containers/Exchange/ExchangeModals/ConfirmModal.js

```diff
@@ -276,6 +276,9 @@ export default class ConfirmModal extends React.Component {
             var data = { sourceAmount, sourceTokenSymbol, destAmount, destTokenSymbol }
             this.props.global.analytics.callTrack("trackCoinExchange", data);
             this.props.global.analytics.callTrack("completeTrade", txHash, "kyber", "swap");
+            
+            const walletName = this.props.account.isOnDAPP ? 'Dapp' : wallet.getWalletName();
```

Comment:
why need to check isOnDAPP ?

Reply:
I thought the wallet name will be `Dapp` if the user is in a Dapp, or else I will remove this check